tests/test_userstory_39.py

```diff
@@ -56,3 +61,75 @@ def test_melding_overlast():
         'object': verblijfsobject['url'],
     })
     assert 'url' in zaak_object
+
+    '''
+    Upload the files with POST /enkelvoudiginformatieobject (DRC)
+    '''
+
+    byte_content = text_file.read() # text_file comes from pytest fixture
+    base64_bytes = b64encode(byte_content)
+    base64_string = base64_bytes.decode('utf-8')
+
+    text_attachment = drc_client.create('enkelvoudiginformatieobject', {
+        'identificatie': uuid.uuid4().hex,
+        'bronorganisatie': '1',
+        'creatiedatum': zaak['registratiedatum'],
+        'titel': 'detailed summary',
+        'auteur': 'test_auteur',
+        'formaat': 'txt',
```

Comment:
IF I remember correctly, MIME-types are used here, so it should be `text/plain` instead of `txt`